client/tests/integration/queries/smart_contract.rs

```diff
@@ -33,7 +33,7 @@ fn live_query_is_dropped_after_smart_contract_end() -> Result<()> {
         client.account.clone(),
         Name::from_str("cursor").unwrap(),
     ))?;
-    let cursor: String = metadata_value.try_into()?;
+    let cursor: String = metadata_value.0;
```

Comment:
this field should be private

Reply:
this is allowed only because our current testing infra includes `transparent_api`. This should and will be disallowed